mautic/mautic
# [UX] Increase export labels specificity

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This update addresses a common frustration for users managing large contact databases. Currently, the ambiguity in export labels leads to unintended bulk exports, wasting time and resources. By distinguishing between page and batch actions, we're giving users more control over their exports.

This PR creates a difference between labels for page actions and batch actions regarding exporting contacts. This aims to solve clients selecting manually batches of 100 contacts to end up exporting the same entire contacts database dozens of times.

Batch:
- Export to CSV
- Export to Excel

Page actions:
- Export all to CSV
- Export all to Excel

![image](https://github.com/user-attachments/assets/ee3fe405-198e-4e64-968e-4db8feb16d53)





---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Click on the page actions dropdown to see the export options
4. Create a new contact and go back to the list
5. Select this contact and click on the batch actions dropdown on the top of the list
6. See the label
